src/app/modules/newsfeed/poster/poster.component.ts

```diff
@@ -75,6 +83,17 @@ export class PosterComponent {
     while ((match = regex.exec(this.meta.message)) !== null) {
       this.tags.push(match[2]);
     }
+
+    // Detect @user tags.
```

Comment:
@markharding, You wanted me to put this snippet in another function, but it literally has to be in `onMessageChange` to receive the JS change event, at least to my knowledge. I can put both it and the regex above it in two separate functions and call them from `onMessageChange`, if you want..